<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLoginDlg</class>
 <widget class="QDialog" name="UserLoginDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>vClient Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_content" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>121</y>
     <width>390</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: bold 14px;
}
QLineEdit{
	border:1px groove #999999;
}
QCheckBox{
	font: bold 13px;
}
QCheckBox::indicator:unchecked {
	image: url(&quot;:/image/resource/image/checkbox_unchecked.png&quot;)
}
QCheckBox::indicator:checked{
	image: url(&quot;:/image/resource/image/checkbox_checked.png&quot;)
}</string>
   </property>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CMyTabWidget" name="tabWidget_login">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>390</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane{border: 0px;}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_normal">
     <attribute name="title">
      <string>       NormalLogin       </string>
     </attribute>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>22</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>UserName:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>22</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_password">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>52</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>52</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_domain">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>81</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Local Domain</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_domain">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>81</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Domain:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_busyDomain">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>106</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox_rempwd">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SavePwd</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_autolg">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-right:0px</string>
         </property>
         <property name="text">
          <string>AutoLogin</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_loading">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_token">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>       TokenLogin       </string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_password_roken">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>81</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_password_token">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>81</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_username_token">
      <property name="geometry">
       <rect>
        <x>37</x>
        <y>22</y>
        <width>84</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>UserName:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username_token">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>22</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_about">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>183</y>
      <width>45</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>45</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>About</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_settings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>183</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>100</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_verticalLine">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>183</y>
      <width>3</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>2</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>3</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/image/resource/image/vertical_line.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyTabWidget</class>
   <extends>QTabWidget</extends>
   <header>cmytabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_login</tabstop>
  <tabstop>tabWidget_login</tabstop>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>comboBox_domain</tabstop>
  <tabstop>pushButton_settings</tabstop>
  <tabstop>lineEdit_username_token</tabstop>
  <tabstop>lineEdit_password_roken</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
